migrations/Version202104130808062141_taoItems.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoItems\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\taoItems\model\user\TaoItemsRoles;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\model\accessControl\func\AclProxy;
+use oat\tao\model\accessControl\func\AccessRule;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202104130808062141_taoItems extends AbstractMigration
```

Comment:
What about have a service for this in `tao-core`?

We can reuse it on at least 2 of your PRs

```
class ApplyRuleService extends ConfigurableService
{
    public function applyRoles(array $rulesByRole): void
    {
        foreach ($rulesByRoleas $role => $rules) {
            foreach ($rules as $rule) {
                AclProxy::applyRule($this->createAclRulesForRole($role, $rule));
            }
        }
    }

    public function revokeRoles(array $roles): void
    {
        foreach ($rulesByRoles as $role => $rules) {
            foreach ($rules as $rule) {
                AclProxy::revokeRule($this->createAclRulesForRole($role, $rule));
            }
        }
    }

    private function createAclRulesForRole(string $role, array $rule): AccessRule
    {
        return new AccessRule(
            AccessRule::GRANT,
            $role,
            $rule
        );
    }
}
```